plotly/tests/test_core/test_tools/test_figure_factory.py

```diff
@@ -686,6 +686,306 @@ def test_datetime_candlestick(self):
+                                                  'y': 'One',
+                                                  'yref': 'y1'},
+                                                 {'font': {'color': '#000000'},
+                                                  'showarrow': False,
+                                                  'text': 'third',
+                                                  'x': 'A',
+                                                  'xref': 'x1',
+                                                  'y': 'Two',
+                                                  'yref': 'y1'},
+                                                 {'font': {'color': '#FFFFFF'},
+                                                  'showarrow': False,
+                                                  'text': 'fourth',
+                                                  'x': 'B',
+                                                  'xref': 'x1',
+                                                  'y': 'Two',
+                                                  'yref': 'y1'},
+                                                 {'font': {'color': '#000000'},
+                                                  'showarrow': False,
+                                                  'text': 'fifth',
+                                                  'x': 'A',
+                                                  'xref': 'x1',
+                                                  'y': 'Three',
+                                                  'yref': 'y1'},
+                                                 {'font': {'color': '#000000'},
+                                                  'showarrow': False,
+                                                  'text': 'sixth',
+                                                  'x': 'B',
+                                                  'xref': 'x1',
+                                                  'y': 'Three',
+                                                  'yref': 'y1'}],
+                                 'xaxis': {'gridcolor': 'rgb(0, 0, 0)', 'side':
+                                           'top', 'ticks': ''},
+                                 'yaxis': {'ticks': '', 'ticksuffix': '  '}}}
+        self.assertEqual(a, expected_a)
+
+
+class TestTable(TestCase):
+
+    def test_fontcolor_input(self):
+
+        # check: PlotlyError if fontcolor input is incorrect
+
+        kwargs = {'table_text': [['one', 'two'], [1, 2], [1, 2], [1, 2]],
+                  'fontcolor': '#000000'}
+        self.assertRaises(PlotlyError,
+                          tls.FigureFactory.create_table, **kwargs)
+
+        kwargs = {'table_text': [['one', 'two'], [1, 2], [1, 2], [1, 2]],
+                  'fontcolor': ['red', 'blue']}
+        self.assertRaises(PlotlyError,
+                          tls.FigureFactory.create_table, **kwargs)
+
+    # def test_simple_table(self):
```

Comment:
Commented just because we changed the default table color- I'm changing the table color in the test and uncommenting now.